### DISCUSSION TITLE
No live preview - Knights Travails, BST, Linked Lists

### DISCUSSION THREAD

**Discussion by JFerrettUK** [TheOdinProject/curriculum#25320] [General]
Hello,

Is there a reason why the computer science projects don't have 'live preview' options when submitting the solutions? Some of them involve visual elements, such as the Knights Travails one for example, while the Linked List uses drivers which could be displayed.

It just seemed like it might be nice for students to show these off similarly to other, earlier projects.

Regards,
James.

**Comment by rlmoser99** [comment]
In the past we did have live previews for these projects and most of the time, the live preview was a rick roll or some other prank. So the team decided it was best to remove this option.
